src/modules/core/components/TokenEditDialog/TokenEditDialog.tsx

```diff
@@ -46,50 +47,74 @@ const MSG = defineMessages({
 });
 
 interface Props {
-  addTokenFn: (address: Address) => Promise<any>;
+  updateTokens: (addresses: Address[]) => Promise<any>;
   cancel: () => void;
   close: () => void;
+  // Colony tokens
   tokens: AnyToken[];
+  // Token list from json file. Not supported on local env
+  tokensList?: AnyToken[];
+  // Colony native token addresss
+  nativeTokenAddress?: Address;
 }
 
 interface FormValues {
   tokenAddress: Address;
   description?: string;
+  tokenAddresses: Address[];
 }
 
 const validationSchema = yup.object({
-  tokenAddress: yup
-    .string()
-    .address()
-    // @todo validate against entering a duplicate address
-    .required(),
-  tokenSymbol: yup.string().max(6),
+  tokenAddress: yup.string().address(),
 });
 
-const TokenEditDialog = ({ addTokenFn, tokens = [], cancel, close }: Props) => {
-  const { formatMessage } = useIntl();
+const TokenEditDialog = ({
+  updateTokens,
+  tokens = [],
+  cancel,
+  close,
+  tokensList = [],
+  nativeTokenAddress,
+}: Props) => {
   const [tokenData, setTokenData] = useState<OneToken | undefined>();
+  const { formatMessage } = useIntl();
 
   const handleTokenSelect = (token: OneToken) => {
     setTokenData(token);
   };
 
   const handleSubmit = useCallback(
```

Comment:
Is there any way to hook into `TokenSelect` (from `ColonyWizard`) in order to be able to invalidate this form _(basically disable the submit button)_ if the token that we're trying to fetch is not valid?

![Screenshot from 2020-12-18 15-27-14](https://user-images.githubusercontent.com/1193222/102621044-b203a980-4147-11eb-9638-fba99da92701.png)

`TokenSelect` throws an error, and if we can hook into, we can keep a local state here and just disable the submit botton based on that.